protocol-designer/src/assets/localization/en/shared.json

```
@@ -57,27 +60,43 @@
       "body6": "Please note that in order to run the updated protocol on your robot successfully, the OT-2 App and robot are required to be updated to version 3.10.0 or higher."
     }
   },
+  "message_exact_labware_match": "This labware is identical to one you have already uploaded.",
+  "message_invalid_json_file": "Protocol Designer only accepts custom JSON labware definitions made with our Labware Creator",
+  "message_not_json": "Protocol Designer only accepts custom JSON labware definitions made with our Labware Creator. This is not a .json file!",
+  "message_only_tiprack": "This labware definition is not a tip rack.",
+  "message_uses_standard_namespace": "This labware definition appears to be an Opentrons standard labware. Please upload only custom labware.",
+  "mismatched": "the new labware has a different arrangement of wells than the definition it is replacing. Clicking the Overwrite button will deselect all wells in any existing steps that use the overwritten definition. You will have to edit each of those steps and re-select the wells.",
   "module": "Module",
   "next": "next",
   "ninety_six_channel": "96-Channel",
   "no-code-required": "The easiest way to automate liquid handling on your Opentrons robot. No code required.",
   "no": "No",
   "none": "None",
+  "not_json": "Incompatible file type",
   "one_channel": "1-Channel",
+  "only_tiprack": "Incompatible file type",
   "opentrons_flex": "Opentrons Flex",
   "opentrons": "Opentrons",
   "ot2": "Opentrons OT-2",
+  "overwrite_labware": "Overwrite labware",
+  "overwrite": "If you intended to replace your previous labware with this new version then proceed with the Overwrite button below.",
   "protocol_designer": "Protocol Designer",
+  "re_export": "If you wish to add this labware then please return to the Labware Creator and export it again with a unique load name and display name.",
   "remove": "remove",
   "right": "Right",
+  "shared_display_name": "Shared display name: ",
+  "shared_load_name": "Shared load name: ",
+  "shares_name": "This labware shares an API Load name and/or a display name with {{customOrStandard}} labware that has already been uploaded.",
   "slot_stack_information": "Slot Stack Information",
   "stagingArea": "Staging area",
   "step_count": "Step {{current}}",
   "step": "Step {{current}} / {{max}}",
-  "trashBin": "Trash bin",
+  "trashBin": "Trash Bin",
+  "uses_standard_namespace": "Standard (non-custom) Opentrons labware",
   "version": "Version # {{version}}",
-  "wasteChute": "Waste chute",
-  "wasteChuteAndStagingArea": "Waste chute and staging area",
+  "warning": "WARNING:",
+  "wasteChute": "Waste Chute",
+  "wasteChuteAndStagingArea": "Waste Chute and staging area",
```

```suggestion
  "wasteChute": "Waste chute",
  "wasteChuteAndStagingArea": "Waste chute with staging area slot",
```
Matches our official names in shared-data: https://github.com/Opentrons/opentrons/blob/59a8f7b887fef475861af5fcc1dd488f207f2dce/shared-data/js/fixtures.ts#L256